src/scribe-plugin-drag-and-drop.js

```diff
@@ -82,6 +82,20 @@ module.exports = function(config) {
     });
 
 
+    helpers.delegate(scribe.el, "dragover", "p", function (event) {
+      if(event.target.className.indexOf(config.style_class) !== -1
+         && event.target.className.indexOf(config.hover_class) === -1) {
```

Comment:
Why not use `classList.has` for this (assuming we don't care for older browser that don't have it, and I don't think we do)? It would be a lot less brittle than substring matching, which doesn't correctly identify classes (e.g. `foo` is in `foobar baz`).